libwebauthn/src/lib.rs

```diff
@@ -34,6 +36,20 @@ pub enum Transport {
     Ble,
 }
 
+#[derive(Debug)]
+pub enum StateUpdate {
```

Comment:
State sounds overly generic. These are UX hints for UV and UP, maybe `UxUpdate`?

Reply:
I was going for overly generic :D But `UxUpdate` does fit really well, indeed!